### DISCUSSION TITLE
Squeezeformer Riva and Checkpoints Support

### DISCUSSION THREAD

**Discussion by piraka9011** [NVIDIA-NeMo/NeMo#4910] [Q&A] [answered]
It seems that Squeezeformer has a similar architecture to Conformer.
Is it (or will it) be supported by Riva?
Also, any chance we'll be getting pre-trained checkpoints?)

**Comment by titu1994** [comment, marked as answer]
We'll release Librispeech pretrained checkpoints for Squeezeformer soon (possibly next month). 

However, I don't know about Riva support. For now, we are focussing our efforts on Conformer as it is more stable and we have a body of work that's built on top of it.
